official/explorer/go.mod

```
@@ -0,0 +1,203 @@
+module github.com/ignite/apps/official/explorer
+
+go 1.21.1
+
+require (
+	github.com/hashicorp/go-plugin v1.5.2
+	github.com/ignite/cli/v28 v28.0.0
```

```suggestion
	github.com/ignite/cli/v28 v28.1.1
```

We should update the other ones too, the latest version changes the ".app" extension to ".ign". The entry in the root gitignore for ".app" could also be removed.